Unix/installbuilder/datafiles/HPUX.data

```diff
@@ -64,3 +70,10 @@ ${{SERVICE_SCRIPT}} start
 %Preuninstall_10
 #include OmiService_funcs
 StopOmiService
+
+%Postuninstall_100
+egrep "^omi:" /etc/group 2>&1 >/dev/null
```

Comment:
Same issue as above. On HP, what, **exactly**, is the ordering of scripts on upgrade?